components/Buttons/examples/ButtonsContentEdgeInsetsExample.m

```diff
@@ -52,7 +52,8 @@ - (void)viewDidLoad {
   self.flatButton.inkColor = [UIColor colorWithWhite:1.0 alpha:0.1];
   self.flatButton.contentEdgeInsets = UIEdgeInsetsMake(64, 64, 0, 0);
   self.raisedButton.contentEdgeInsets = UIEdgeInsetsMake(0, 0, 64, 64);
-  self.floatingActionButton.contentEdgeInsets = UIEdgeInsetsMake(40, 40, 0, 0);
+  [self.floatingActionButton setContentEdgeInsets:UIEdgeInsetsMake(40, 40, 0, 0)
+                                         forShape:MDCFloatingButtonShapeDefault];
```

Comment:
There would be breaking changes by making properties unavailable:
* minimumSize
* maximumSize
* contentEdgeInsets
* hitAreaInsets

The alternative is not much better, however, since "shape" does not have a true "default" value the way UIControlState does.

Reply:
Thanks for acknowledging that we are NS_UNAVAILABLE 5 properties. you missed:
- elevationForState: